integration/time_drift/time_drift.bats

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# This test will make sure that guest time
+# is synced with the host
+
+set -e
+
+load "${BATS_TEST_DIRNAME}/../../lib/common.bash"
+
+# Save guest time
+GUEST_TMP_FILE=$(mktemp guest.XXXXX)
+# Save host time
+HOST_TMP_FILE=$(mktemp host.XXXXX)
+
+function remove_tmp_files() {
+	rm "${GUEST_TMP_FILE}" "${HOST_TMP_FILE}"
+}
+
+trap remove_tmp_files EXIT
+
+setup() {
+	clean_env
+
+	# Check that processes are not running
+	run check_processes
+	echo "$output"
+	[ "$status" -eq 0 ]
+}
+
+@test "Verify guest time is synced with the host" {
+	CONTAINER_NAME="test"
+	IMAGE="ubuntu"
+	PAYLOAD="tail -f /dev/null"
+
+	docker run -d --name "${CONTAINER_NAME}" --runtime "${RUNTIME}" "${IMAGE}" sh -c "${PAYLOAD}"
+
+	# Get host and guest time
+	(docker exec "${CONTAINER_NAME}" date > "${GUEST_TMP_FILE}"; date > "${HOST_TMP_FILE}") | parallel
+
+	guest_time=$(cat "${GUEST_TMP_FILE}")
+	host_time=$(cat "${HOST_TMP_FILE}")
+
+	[ "$host_time" == "$guest_time" ]
```

Comment:
So, I think this will be 'accurate to the second', yes?
I'm not going to be surprised if maybe 1 in 100 runs of this test happen to hit the rollover boundary of the second between the host and guest, and fail... Two things then:
- can we add an echo/info output here in the case of failure, and show the two times - then we get some clues to the failure? (not sure if `bats` does that already etc.??)
- be prepared to maybe rewrite the test in the future to grab the time in nanoseconds or similar and set an 'acceptable difference' value, to do a bit more of a fuzzy check...